mage.ts

```diff
@@ -2415,7 +2415,7 @@ declare namespace mage {
              *
              * @memberOf State
              */
-            emit<T>(actorId: string | string[], eventName: string | number, data: T): void;
+            emit<T>(actorId: string | string[], eventName: string | number, data: T, configuration: mage.core.IStateEmitConfig): void;
```

Comment:
You might also want to add the `emitEvents` definition since I've seen it used in quite a few projects